Apromore-Custom-Plugins/Log-Importer-Logic/src/main/java/org/apromore/service/logimporter/utilities/ImporterStringUtils.java

```diff
@@ -94,7 +97,12 @@ public static ColumnType getColumnType(final List<String> stringList) {
         temp.put(ColumnType.DOUBLE, 0);
         temp.put(ColumnType.TIMESTAMP, 0);
 
-        stringList.forEach(st -> {
+        List<String> filteredList = stringList.stream()
+            .filter(Objects::nonNull)
```

Comment:
Could combine both using StringUtils.isEmpty(). This will test for both the null and empty strings